lib.rs

```diff
@@ -343,7 +343,7 @@ fn process_toml(cargo_toml: &str, args: &Args) -> Result<String, String> {
                     }
                 } else if current_table != "features" {
                     return Err(format!(
-                        "Comment cannot be associated with a feature: '{}'",
+                        r#"Comment cannot be associated with a feature: "{}""#,
                         current_comment
```

Comment:
Maybe the error would look better if we trim the whitespace out of current_comment?

Reply:
yes, see the next commit I did
has the exact same idea